test/manual/atlas_connectivity.test.ts

```diff
@@ -0,0 +1,67 @@
+import { LEGACY_HELLO_COMMAND, MongoClient } from '../mongodb';
+
+/**
+ * ATLAS_CONNECTIVITY env variable is JSON
+ * Here's some typescript describing the shape:
+ *
+ * ```typescript
+ * interface AtlasConnectivity {
+ *  [atlasDeployment: string]: [normalUri: string, srvUri: string]
+ * }
+ * ```
+ *
+ * It should be an object with descriptive strings about the deployment type and version (i.e. sharded_cluster_3_4)
+ * that map to a two string tuple that are the normal URI and SRV URI, order doesn't matter, but it should be that order.
+ */
+
+describe('Atlas Connectivity', function () {
+  if (process.env.ATLAS_CONNECTIVITY == null) {
+    console.error(
+      'skipping atlas connectivity tests, ATLAS_CONNECTIVITY environment variable is not defined'
+    );
+
+    return;
+  }
```

Comment:
Should I delete this? if we accidentally removed the env var then these tests would silently skip right?

Reply:
Agreed.  

Did these tests live in `tests/integration` at any point?  I'm wondering if is here to prevent failures on regular integration runs.